src/components/editor-page/editor-pane/table-extractor.test.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { isTable } from './table-extractor'
+
+describe('isTable detection', () => {
```

Comment:
Where are the tests for `extractTable` ?

Reply:
Added